python/humbug/report.py

```diff
@@ -119,6 +124,44 @@ def system_tags(self) -> List[str]:
 
         return tags
 
+    def _apply_blacklist_to_parameters(
```

Comment:
@kompotkot : I suggest making a `blacklist_fn` argument to `__init__` in which user can specify whatever blacklist function they want.

Instead of using this particular implementation of `_apply_blacklist_to_parameters`, the reporter will be able to use any user-specified function to omit parameters.

For example, a regex-match condition for specific expressions (like `token`).